projects/components/src/select/select.component.scss

```diff
@@ -84,8 +84,7 @@
 }
 
 .select-content {
-  @include dropdown();
-  min-width: 120px;
+  @include dropdown(4px, 0, 1px 0 4px 0);
```

Comment:
Why does the select have different styling than other dropdowns? It doesn't seem like it should be different/parameterizable.